pkg/maps/ipmasq/ipmasq.go

```diff
@@ -121,6 +131,10 @@ func (*IPMasqBPFMap) Dump() ([]net.IPNet, error) {
 	return cidrs, nil
 }
 
+func (*IPMasqBPFMap) Dump() ([]net.IPNet, error) {
```

Comment:
Still used? Also no comment on exported method. 

Perhaps a larger discussion here but this sounds like prime candidates for generics. 
There is very little use for this type as shown below as since this is just a `marker` type that just keep checking what kind of ips are at play and whether the correct matching configs are set. 
> Note: in this case we need an instance that is passed in on the fly.

Reply:
Still used, in the agent. The method didn't have a comment so far and I didn't think of adding one; but I can do, for both.

As for the generics, it makes sense, but have we been using them so far in the code base? This PR is a set of fixes, I'd as well avoid moving things too much or introducing generics in here.